docs-source/content/userguide/managing-operators/installation/_index.md

```diff
@@ -109,7 +109,9 @@ Then install the 3.x operator using the [installation](#install-the-operator-hel
 The following instructions will be applicable to upgrade operators within the 3.x release family
 as additional versions are released.
 
-To upgrade the operator, use the `helm upgrade` command. When upgrading the operator,
+To upgrade the operator, use the `helm upgrade` command. Make sure that the
+`weblogic-kubernetes-operator` repository in your local machine is at the
+operator release that you are upgrading to. When upgrading the operator,
```

Comment:
at the operator release that you are upgrading to -> at the operator release to which you are upgrading.

Reply:
updated. thanks!